### DISCUSSION TITLE
Can I Configure Language Server Options Specific to a Workspace?

### DISCUSSION THREAD

**Discussion by zicklag** [helix-editor/helix#8510] [Q&A] [answered]
My use-case is setting rust-analyzer to use a `wasm32-unknown-unknown` target, but only for the workspace that is targeting web, not all of them. Is this possible?

**Comment by pascalkuthe** [comment, marked as answer]
Helix supports overwriting all co fig options in a workspace specific config file (`.helix/languages.toml` and `.helix/config.toml`)

**Reply by zicklag** [reply]
Awesome, that's perfect, thanks!
